vsintegration/src/FSharp.LanguageService/ProjectSitesAndFiles.fs

```diff
@@ -49,10 +49,14 @@ type private IHaveCheckOptions =
 /// Convert from FSharpProjectOptions into IProjectSite.         
 type private ProjectSiteOfScriptFile(filename:string, referencedProjectFileNames, checkOptions : FSharpProjectOptions) = 
     interface IProjectSite with
-        override this.SourceFilesOnDisk() = checkOptions.SourceFiles
-        override this.DescriptionOfProject() = sprintf "Script Closure at Root %s" filename
-        override this.CompilerFlags() = checkOptions.OtherOptions
-        override this.ProjectFileName() = checkOptions.ProjectFileName
+        override this.Description = sprintf "Script Closure at Root %s" filename
+        override this.CompilationSourceFiles = checkOptions.SourceFiles
+        override this.CompilationOptions = checkOptions.OtherOptions
+        override this.CompilationReferences = 
+             checkOptions.OtherOptions 
+             |> Array.choose (fun flag -> if flag.StartsWith("-r:") then Some flag.[3..] else None) 
```

Comment:
Can references not start with `/r:` too?

Reply:
Yes, thx.